assets/prefabs/items/Waterskin.prefab

```diff
@@ -8,10 +8,11 @@
     "name": "Waterskin"
   },
   "FluidContainerItem": {
-      "maxVolume": 500
-  },
-  "MaterialFluidItemContainer": {
-      "fluidIcon": "ManualLabor:ManualLabor#WoodenCupFluid"
+    "maxVolume": 500,
+    "emptyTexture": "emptyWaterskin",
+    "textureWithHole": "ManualLabor:ManualLabor#Waterskin",
```

Comment:
texture with hole -> texture for when the item is filled>
https://github.com/Terasology/Fluid/blob/00e18ab0acd8adfbe54847aeab4c4c90ae07f8e1/src/main/java/org/terasology/fluid/component/FluidContainerItemComponent.java#L45-L46

the fluidMin/SizePerc s are required as they are checked if null when setting the icon for filled item.
https://github.com/Terasology/Fluid/blob/00e18ab0acd8adfbe54847aeab4c4c90ae07f8e1/src/main/java/org/terasology/fluid/system/FluidClientSystem.java#L103-L106

MaterialFluidItemContainer is not needded as it is only used in the system which adds a tint overlay in items (water tint in woodenCup and Bucket)